### ISSUE TITLE
Inconsistent Comment Spacing

### ISSUE THREAD

**Issue by NeoKenshinX** [dotnet/format#612]
Regarding single line comments at least the space after // should be formatted as mentioned in the coding conventions: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#commenting-conventions

**Before Formatting**
```
//this comment is not correctly formatted
// this comment is correctly formatted
```

**Actual Behavior**
```
//this comment is not correctly formatted
// this comment is correctly formatted
```

**Expected Behavior**
```
// this comment is not correctly formatted
// this comment is correctly formatted
```

**Comment by jmarolf** [comment]
@JoeRobich I think this should move to roslyn as we'd want this to be a formatting rule that the IDE follows as well

**Comment by sharwell** [comment]
I don't think we can make this change in Roslyn or enable it by default in dotnet/format. Too many users would experience breaking behavior changes when attempting to comment out code blocks.

**Comment by jmarolf** [comment]
> Too many users would experience breaking behavior changes when attempting to comment out code blocks.

If we were to add this to roslyn I would argue that the comment/uncomment behavior should respect this setting.
